app/controllers/status_messages_controller.rb

```diff
@@ -78,7 +78,8 @@ def create
         current_user.dispatch_post(photo)
       end
 
-      current_user.participate!(@status_message)
+      participation = current_user.participate!(@status_message)
+      participation.update!(count: participation.count.next)
```

Comment:
I don't quite follow, don't you do this in `find_or_create_participation!` already?